prototype_source/fx_graph_mode_ptq_static.rst

```diff
@@ -185,252 +187,223 @@ Download the `torchvision resnet18 model <https://download.pytorch.org/models/re
+3. Set model to eval mode
+-------------------------
 For post training quantization, we'll need to set model to eval mode.
 
 .. code:: python
 
-    model_to_quantize.eval()    
-
-  
-4. Specify how to quantize the model with ``qconfig_dict``    
-----------------------------------------------------------    
-  
-.. code:: python  
-  
-  qconfig_dict = {"" : default_qconfig}   
-  
-We use the same qconfig used in eager mode quantization, ``qconfig`` is just a named tuple    
-of the observers for activation and weight. ``qconfig_dict`` is a dictionary with the following configurations:   
-  
-.. code:: python  
-  
-  qconfig = { 
-      "" : qconfig_global,
-      "sub" : qconfig_sub,    
-      "sub.fc" : qconfig_fc,  
-      "sub.conv": None    
-  }   
-  qconfig_dict = {    
-      # qconfig? means either a valid qconfig or None 
-      # optional, global config   
-      "": qconfig?,   
-      # optional, used for module and function types  
-      # could also be split into module_types and function_types if we prefer 
-      "object_type": [    
-          (torch.nn.Conv2d, qconfig?),    
-          (torch.nn.functional.add, qconfig?),    
-          ...,    
-      ],  
-      # optional, used for module names   
-      "module_name": [    
-          ("foo.bar", qconfig?)   
-          ...,    
-      ],  
-      # optional, matched in order, first match takes precedence  
-      "module_name_regex": [  
-          ("foo.*bar.*conv[0-9]+", qconfig?)  
-          ...,    
-      ],  
-      # priority (in increasing order): global, object_type, module_name_regex, module_name   
-      # qconfig == None means fusion and quantization should be skipped for anything  
-      # matching the rule 
-      
-      # **api subject to change** 
-      # optional: specify the path for standalone modules 
-      # These modules are symbolically traced and quantized as one unit   
-      # so that the call to the submodule appears as one call_module  
-      # node in the forward graph of the GraphModule  
-      "standalone_module_name": [ 
-          "submodule.standalone"  
-      ],  
-      "standalone_module_class": [    
-          StandaloneModuleClass   
-      ]   
-  }   
-  
-Utility functions related to ``qconfig`` can be found in the `qconfig <https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/qconfig.py>`_ file.
+    model_to_quantize.eval()
+
+
+4. Specify how to quantize the model with ``QConfigMapping``
+----------------------------------------------------------
+
+.. code:: python
+
+  qconfig_mapping = QConfigMapping.set_global(default_qconfig)
+
+We use the same qconfig used in eager mode quantization, ``qconfig`` is just a named tuple
+of the observers for activation and weight. ``QConfigMapping`` contains mapping information from ops to qconfigs:
+
+.. code:: python
+
+  qconfig_mapping = (QConfigMapping()
+      .set_global(qconfig?)  # qconfig? means either a valid qconfig or None
```

Comment:
maybe let's call it `qconfig_opt` or something, I feel `qconfig?` is a bit weird to write in code